### ISSUE TITLE
Distance to position, broken?

### ISSUE THREAD

**Issue by MephestoKhaan** [wrld3d/unity-api#84]
I think the "distance" calculations are not working when the map is scaled. If I mesure the distance from the camera to the center of the map I get a value X, but if I get the distance from the camera to something that is actually closer but not centered, the distance is sometimes higher.

I am doing this
```
LatLongAltitude currentCoords = Api.Instance.CameraApi.WorldToGeographicPoint(Camera.main.transform.position, Camera.main);
double distance = LatLong.EstimateGreatCircleDistance(targetCoords, currentCoords.GetLatLong());

if (distance > jumpDistanceThreshold)
{ 
// recenter map
}
```

And if I scroll manually, without centering the map, to `targetCoords` until I am sure I am at < jumpDistanceThreshold  and then run this code the distance is *huge*

**Comment by MephestoKhaan** [comment]
I tried something similar converting the `targetCoords` to Vector3 with the `CameraAPI` and I see similar results.

**Comment by johnbell84** [comment]
Hi MephestoKhaan,

Thanks for trying WRLD, and thanks very much for the issue report.  

We're investigating some issues to do with applying Unity transforms to our maps at the moment.  We'll update this issue with more information when we can, but in the meantime, I'd suggest leaving the maps unscaled as a workaround, if that's possible for you.

I hope this helps, best of luck with your project.

Kind regards,

John

**Comment by MephestoKhaan** [comment]
Thanks! Impossible at the moment sadly as I am using for AR purposes and
really need it scaled. Do you have an ETA?

**Comment by deadbytedawn** [comment]
@MephestoKhaan You could try turning on the terrain collision and cast a sphere: https://docs.unity3d.com/ScriptReference/Physics.SphereCast.html

**Comment by ghost** [comment]
Believe this is now resolved in release v0.8
